.github/workflows/validate.yaml

```diff
@@ -0,0 +1,29 @@
+name: Lint and Test Charts
+on:
+  pull_request:
+    branches:
+      - dev-v2.6
+
+jobs:
+  validate-chart-questions:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Validate all charts have a questions file
+        run: >
+          chart_dirs=$(find charts -type d -mindepth 2 -maxdepth 2)
+
+          for chart in $chart_dirs; do
+            if [[ ! -f $chart/questions.yaml ]] && [[ ! -f $chart/questions.yml ]]; then
+              echo "$chart is missing a questions chart file"
+              exit 1
+            fi
+          done
```

Comment:
wondering if both of these scripts should be located under `./scripts` instead. What are your thoughts?

Reply:
seems like this was the same concern brought up in https://github.com/rancher/system-charts/pull/504#discussion_r760588402. @paynejacob @PennyScissors if you both agree still, then I'm find with skipping this as well.

But I think we still need a comment for readability.